src/agent/onefuzz-task/src/tasks/generic/input_poller.rs

```diff
@@ -251,6 +251,7 @@ impl<M> InputPoller<M> {
         if result.is_err() {
             // We must maintain a valid state, and we can logically recover from
             // any failed action or invalid transition.
+            log::error!("State Transition Failed. Resetting state to 'Ready.'");
```

Comment:
This will only log locally, not to telemetry.

We want:
```suggestion
            error!("State Transition Failed. Resetting state to 'Ready.'");
```